k8s/jenkins/scripts/slack_notifier.groovy

```diff
@@ -0,0 +1,69 @@
+#!groovy
+import com.cloudbees.jenkins.plugins.sshcredentials.impl.*
+import com.cloudbees.plugins.credentials.*
+import com.cloudbees.plugins.credentials.common.*
+import com.cloudbees.plugins.credentials.domains.Domain
+import com.cloudbees.plugins.credentials.impl.*
+import hudson.util.Secret
+import java.nio.file.Files
+import jenkins.model.Jenkins
+import net.sf.json.JSONObject
+import org.jenkinsci.plugins.plaincredentials.impl.*
+
+def dataList = [:]
+def file = new File("/usr/share/jenkins/ref/init.groovy.d/slack.config")
+
+def text
+def key
+def value
+
+file.eachLine { line ->
+
+if (line.trim().size() == 0) {
+  return null
+
+} else {
+
+  text = line.split("=")
+  key=text[0] 
+  value=text[1]
+  valueSub=value.substring(1,value.length()-1)
+  dataList[key]=valueSub
+  }
+}
+
+def slackCredentialParameters = [
+  description:  'Slack integration token',
+  id:           'slack-token',
+  secret:       "${dataList.token}"
+]
+ 
+def slackParameters = [
+  slackBaseUrl:             "${dataList.slackBaseUrl}",
+  slackBotUser:             'false',
+  slackRoom:                '#ada_nbcustomer_algo',
```

Comment:
Why do you hardcode name of room?

Reply:
because it is default room value for the current task like #general channel. it can be changed in jenkins configuration but should be configured automatically on start up of small jenkins. All other values are listed in vault secrets

Reply:
Your position is incorrect. Value should be taken from profile